### PR TITLE
fix(VSimpleCheckbox) :class does not applying

### LINKED ISSUE #11807: [Bug Report] (VSimpleCheckbox) :class property does not add class
### Environment
**Vuetify Version:** 2.1.15
**Vue Version:** 2.6.10
**Browsers:** Edge 84.0.522.28, Google Chrome
**OS:** Windows 10, Mac OSX

### Steps to reproduce
Add `:class` property to v-simple-checkbox

### Expected Behavior
`:class` property adds class to checkbox

### Actual Behavior
`:class` property does not add class to checkbox

### Reproduction Link
<a href="https://codepen.io/mrrefactoring/pen/VweXyaY" target="_blank">https://codepen.io/mrrefactoring/pen/VweXyaY</a>

### PR DESCRIPTION
**Pull request by MrRefactoring**
## Description

The `:class` prop was not adds class to `v-simple-checkbox` component.

## Motivation and Context

fixes #11807 

## How Has This Been Tested?

visually

## Markup:

<details>

```vue
<script type="text/x-template" id="app-template">
  <v-app>
    <v-container>
      <v-simple-checkbox :class="{ 'ripple-color': true }" />
    </v-container>
  </v-app>
</script>

<div id="app"></div>

<style>
  .ripple-color .v-ripple__container {
    background: red;
  }
</style>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import Colorable from '../../mixins/colorable'
 import Themeable from '../../mixins/themeable'
 
 // Utilities
-import { mergeListeners } from '../../util/mergeData'
+import { mergeListeners, mergeClasses } from '../../util/mergeData'
```
Comment (johnleider) [inline packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts:13]: move under line 14, break into new lines and order alphabetically please:

https://www.notion.so/vuetify/Components-d863ad73563d43609c434f92b3e25f08#9127c5ebc666490eaa82bfe633e56dcb

Comment (MrRefactoring) [reply packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts]: Resolved

File: packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts [hunk 2]
```diff
@@ -80,7 +80,7 @@ export default Vue.extend({
 
     return h('div', {
       ...data,
```
Comment (KaelWD) [inline packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts:82]: Is there a reason we aren't just using mergeData for the whole lot?

Comment (MrRefactoring) [reply packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts:82]: I think we can do it. It is not clear why it was not done so initially, therefore, I presented such a solution. Should I remake it into what you proposed?

Comment (KaelWD) [reply packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts:82]: @jacekkarczmarczyk did it in #11358, maybe he remembers

Comment (jacekkarczmarczyk) [reply packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts:82]: Becase in that PR there were only listeners to merge, if there are more thing to merge `mergeData` can be used as well

File: packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts [hunk 3]
```diff
@@ -78,18 +78,19 @@ export default Vue.extend({
...
+
+            if (data.on && data.on.input && !props.disabled) {
+              wrapInArray(data.on.input).forEach(f => f(!props.value))
+            }
+          },
         },
-      }, listeners),
-    }, children)
+      },
+      listeners), children)
```
Comment (KaelWD) [inline packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts:94]:
```diff
        },
-      },
-      listeners), children)
+      }), children)
  },
```

**Pull request merged**